input/fsh/Profiles clinical datapoints/ListofAliases.fsh

```diff
@@ -0,0 +1,23 @@
+// Profiles HL7 FHIR
+Alias: StandardBodyWeight = http://hl7.org/fhir/StructureDefinition/bodyweight 
+Alias: StandardBodyHeight = http://hl7.org/fhir/StructureDefinition/bodyheight
+
+// CodeSystems:
+Alias: SCT = http://snomed.info/sct
+Alias: LNC = http://loinc.org
+Alias: AJCC = http://cancerstaging.org
+Alias: NullFlavor = http://terminology.hl7.org/CodeSystem/v3-NullFlavor
+Alias: RelationshipStatusCS = http://terminology.hl7.org/CodeSystem/v3-MaritalStatus
+Alias: RaceCS = http://terminology.hl7.org/CodeSystem/v3-Race 
+Alias: GenderCS = http://hl7.org/fhir/administrative-gender
+Alias: EthnicityCS = http://terminology.hl7.org/CodeSystem/v3-Ethnicity
+Alias: CountryCS = urn:iso:std:iso:3166
+Alias: ObservationStatusCS = http://hl7.org/fhir/observation-status
+Alias: ConditionStatusCS = http://terminology.hl7.org/CodeSystem/condition-clinical
+Alias: UCUM = http://unitsofmeasure.org
+
+// Valuesets
+Alias: EthnicityVS = http://terminology.hl7.org/ValueSet/v3-Ethnicity
+// http://hl7.org/fhir/us/core/ValueSet/detailed-ethnicity --> other option
+Alias: CountryVS = 	http://hl7.org/fhir/ValueSet/iso3166-1-3
+Alias: RaceVS = http://terminology.hl7.org/ValueSet/v3-Race // is not working, we need something else
```

Comment:
The valueSet for race gives an error in the QA report and doesn't seem to work. We need to discuss with ICHOM which valueset (and codeSystems) we'd like to use here.